code/modules/power/apc.dm

```diff
@@ -787,6 +787,49 @@
 	. = ..()
 	if(.)
 		return
+
+	var/mob/living/carbon/human/H = user
+	if(istype(H))
+		var/datum/species/ethereal/eth_species = H.dna?.species
+		if(istype(eth_species) && H.a_intent == INTENT_HARM)
+			if(cell.charge <= (cell.maxcharge / 2)) // if charge is under 50% you shouldnt drain it
+				to_chat(H, "<span class='warning'>The APC doesn't have much power, you probably shouldn't drain any.</span>")
```

Comment:
do you think that an antag ethereal character would care about draining all of the power out of an APC? I feel like there should be a better explanation for this than "your character (who just put down a power sink) doesn't WANT to drain this APC dry because reasons" (maybe "your character physically CAN'T drain this APC dry"? or just remove this limitation, it's kind of arbitrary).

Reply:
i feel as if its fine, but I _could_ change the explanation to something more apc function wise, like: “the APC’s safeties prevent you from draining any power!” Or something